src/components/AddDependencySearchResult/AddDependencySearchResult.js

```diff
@@ -132,7 +132,7 @@ class AddDependencySearchResult extends PureComponent<Props> {
       <Wrapper>
         <Header>
           <Title>
-            <ExternalLink href={npmLink}>
+            <ExternalLink display={'inline'} href={npmLink}>
```

Comment:
Nit: This could just be `display="inline"`. Doesn't really matter, but it follows convention for strings to use the short-hand.

Reply:
TIL!